src/legacy/core_plugins/timelion/public/directives/saved_object_finder.js

```diff
@@ -165,7 +165,7 @@ module
         };
 
         self.getLabel = function () {
-          return _.words(self.properties.nouns).map(_.capitalize).join(' ');
+          return _.words(self.properties.nouns).map(_.upperFirst).join(' ');
```

Comment:
Why the change here? Looks like capitalize still exists in 4.x and this would result in different behavior.

Reply:
I've changed this on purpose exactly to match the old behaviour on lodash@3. 
If you check here on that example https://jsbin.com/runimajetu/edit?html,js,console,output `upperFirst` is the only one that provides the same output between lodash@3 and lodash@4.

Reply:
Ah, good find - I missed that.